scripts/generate-react-exports.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs-extra');
+const prettier = require('prettier');
+
+function writeBaklavaReactFile(fileContentParts) {
+  let fileContentText = `
+    /* eslint-disable @typescript-eslint/ban-ts-comment */
+    // @ts-nocheck
+    import React from 'react';
```

Comment:
Currently "react" is bundled in our `baklava-react.js` file because of this import. Please add `external: ['react'],` to build js to exclude react from this file in the bundle.